Downloading Apollo Error

Hi everyone, 

I'm very new to Apollo and command line installation, so please bear with me. I am trying to download Apollo using the quick-start guide here: http://webapollo.readthedocs.io/en/latest/Apollo2Build.html

I have downloaded everything until the step that says "Verify install requirements" by using the command line ./apollo run-local. I receive an error that says:

"Grails not found using grailsw
Gradle not found using gradlew
Defining custom 'build' task when using the standard Gradle lifecycle plugins has been deprecated and is scheduled to be removed in Gradle 3.0
Defining custom 'clean' task when using the standard Gradle lifecycle plugins has been deprecated and is scheduled to be removed in Gradle 3.0
No log file found in classpath.
final JBrowse settings [git:[url:https://github.com/GMOD/jbrowse, branch:master, alwaysPull:true, alwaysRecheck:true], plugins:[WebApollo:[included:true], NeatHTMLFeatures:[included:true], NeatCanvasFeatures:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
final plugins [WebApollo:[included:true], NeatHTMLFeatures:[included:true], NeatCanvasFeatures:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]
Copying apollo plugin
handling jbrowse release [git:[url:https://github.com/GMOD/jbrowse, branch:master, alwaysPull:true, alwaysRecheck:true], plugins:[WebApollo:[included:true], NeatHTMLFeatures:[included:true], NeatCanvasFeatures:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
handling jbrowse [git:[url:https://github.com/GMOD/jbrowse, branch:master, alwaysPull:true, alwaysRecheck:true], plugins:[WebApollo:[included:true], NeatHTMLFeatures:[included:true], NeatCanvasFeatures:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
:evaluateJBrowseConfigs UP-TO-DATE
:installJBrowse
installing jbrowse [git:[url:https://github.com/GMOD/jbrowse, branch:master, alwaysPull:true, alwaysRecheck:true], plugins:[WebApollo:[included:true], NeatHTMLFeatures:[included:true], NeatCanvasFeatures:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
checkout out branch master
Out: Already on 'master'
Your branch is behind 'origin/master' by 58 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)

Pulling https://github.com/GMOD/jbrowse
git pull /Users/mimiyxu/Apollo/jbrowse-download
Out: Updating 1bac96a..61bc96f
Fast-forward
 .travis.yml                                        |    9 +-
 ChangeLog.md                                       |    2 +-
 README.md                                          |    6 +-
 apollo                                             |   68 +-
 application.properties                             |    5 +-
 build.gradle                                       |   69 +-
 build.xml                                          |   11 +-
 client/apollo/js/View/Dialog/Help.js               |    2 +-
 client/apollo/js/View/Track/AnnotTrack.js          |  489 +++++--
 client/apollo/js/View/Track/AnnotTrackSocket.js    |  248 ----
 client/apollo/js/View/Track/SequenceTrack.js       |    3 +-
 client/apollo/js/main.js                           |    6 +-
 docs/ChadoExport.md                                |    6 +-
 docs/Configure.md                                  |    2 +-
 docs/TestScript.md                                 |    2 +-
 .../suite2/jmeter-dev-stress-test-notes.md         |    4 +-
 docs/index.rst                                     |    5 +-
 .../examples/groovy/SampleFeatures.groovy          |   11 +-
 docs/web_services/examples/groovy/get_gff3.groovy  |    1 -
 grails-app/conf/BootStrap.groovy                   |    8 +
 grails-app/conf/BuildConfig.groovy                 |    7 +-
 grails-app/conf/Config.groovy                      |    8 +-
 grails-app/conf/DataSource.groovy                  |    9 +-
 grails-app/conf/UrlMappings.groovy                 |   21 +-
 .../conf/org/bbop/apollo/SecurityFilters.groovy    |   14 +-
 .../bbop/apollo/AnnotationEditorController.groovy  |   53 +-
 .../org/bbop/apollo/AnnotatorController.groovy     |   57 +-
 .../org/bbop/apollo/FeatureEventController.groovy  |    3 +-
 .../org/bbop/apollo/GroupController.groovy         |  231 ++-
 .../org/bbop/apollo/HomeController.groovy          |    6 +-
 .../org/bbop/apollo/IOServiceController.groovy     |  159 +-
 .../org/bbop/apollo/JbrowseController.groovy       |  198 +--
 .../org/bbop/apollo/OrganismController.groovy      |   11 +-
 .../org/bbop/apollo/SequenceController.groovy      |   17 +-
 .../org/bbop/apollo/UserController.groovy          |   67 +-
 .../domain/org/bbop/apollo/Preference.groovy       |    6 +
 .../org/bbop/apollo/UserOrganismPreference.groovy  |    3 +-
 ...angelog-2.0.0.groovy => changelog-2_0_0.groovy} |    0
 ...angelog-2.0.1.groovy => changelog-2_0_1.groovy} |    0
 ...angelog-2.0.2.groovy => changelog-2_0_2.groovy} |    0
 ...angelog-2.0.3.groovy => changelog-2_0_3.groovy} |    0
 grails-app/migrations/changelog.groovy             |    6 +-
 .../org/bbop/apollo/AnnotatorService.groovy        |    9 +-
 .../org/bbop/apollo/ChadoHandlerService.groovy     |   18 +-
 .../services/org/bbop/apollo/ExonService.groovy    |    6 +-
 .../org/bbop/apollo/FeatureEventService.groovy     |   26 +-
 .../bbop/apollo/FeatureRelationshipService.groovy  |   67 +-
 .../services/org/bbop/apollo/FeatureService.groovy |  413 +++++-
 .../apollo/NonCanonicalSplitSiteService.groovy     |   57 +-
 .../org/bbop/apollo/PermissionService.groovy       |  214 ++-
 .../org/bbop/apollo/PreferenceService.groovy       |  183 ++-
 .../org/bbop/apollo/RequestHandlingService.groovy  |  152 +-
 grails-app/views/annotator/index.gsp               |    1 +
 grails-app/views/auth/login.gsp                    |   93 +-
 .../views/jbrowse/chooseOrganismForJbrowse.gsp     |    2 +-
 grails-app/views/layouts/login.gsp                 |  107 ++
 grailsw                                            |    8 +-
 grailsw.bat                                        |    8 +-
 restapidoc.json                                    | 1538 ++++++++++----------
 sample-h2-apollo-config.groovy                     |   37 +-
 sample-mysql-apollo-config.groovy                  |   46 +-
 sample-postgres-apollo-config.groovy               |   92 +-
 .../chado-schema-with-ontologies.sql.gz            |  Bin
 .../bbop/apollo/history/FeatureOperation.groovy    |    3 +-
 src/gwt/org/bbop/apollo/gwt/client/Annotator.java  |   77 +-
 .../org/bbop/apollo/gwt/client/AnnotatorPanel.java |   19 +-
 src/gwt/org/bbop/apollo/gwt/client/GroupPanel.java |   13 +-
 .../org/bbop/apollo/gwt/client/GroupPanel.ui.xml   |    2 +-
 src/gwt/org/bbop/apollo/gwt/client/MainPanel.java  |   44 +-
 .../apollo/gwt/client/ReferenceSequenceOracle.java |    9 +-
 .../org/bbop/apollo/gwt/client/resources/Table.css |    3 +-
 .../bbop/apollo/gwt/client/rest/RestService.java   |    8 +
 .../apollo/gwt/shared/ClientTokenGenerator.java    |   30 +
 .../bbop/apollo/gwt/shared/FeatureStringEnum.java  |    2 +
 src/templates/war/web.xml                          |   82 ++
 test/config/mysql.travis                           |    9 +
 test/config/postgres.travis                        |    6 +-
 .../bbop/apollo/CdsServiceIntegrationSpec.groovy   |    9 +-
 .../ChadoHandlerServiceIntegrationSpec.groovy      |    7 +-
 ...eatureRelationshipServiceIntegrationSpec.groovy |   29 +
 .../apollo/FeatureServiceIntegrationSpec.groovy    |  135 ++
 .../Gff3HandlerServiceIntegrationSpec.groovy       |   10 +-
 .../apollo/OverlapperServiceIntegrationSpec.groovy |   30 +-
 .../RequestHandlingServiceIntegrationSpec.groovy   |  340 ++++-
 .../apollo/SequenceServiceIntegrationSpec.groovy   |    4 +-
 .../org/bbop/apollo/PermissionServiceSpec.groovy   |    6 +-
 web-app/js/restapidoc/restapidoc.json              | 1538 ++++++++++----------
 ...-2.4.3.jar => grails-wrapper-runtime-2.4.5.jar} |  Bin 6185 -> 6185 bytes
 wrapper/grails-wrapper-runtime-2.5.4.jar           |  Bin 0 -> 6110 bytes
 wrapper/grails-wrapper.properties                  |    2 +-
 wrapper/springloaded-1.2.0.RELEASE.jar             |  Bin 412416 -> 0 bytes
 wrapper/springloaded-1.2.3.RELEASE.jar             |  Bin 0 -> 419159 bytes
 wrapper/springloaded-1.2.4.RELEASE.jar             |  Bin 0 -> 427010 bytes
 93 files changed, 4135 insertions(+), 3192 deletions(-)
 delete mode 100644 client/apollo/js/View/Track/AnnotTrackSocket.js
 rename grails-app/migrations/{changelog-2.0.0.groovy => changelog-2_0_0.groovy} (100%)
 rename grails-app/migrations/{changelog-2.0.1.groovy => changelog-2_0_1.groovy} (100%)
 rename grails-app/migrations/{changelog-2.0.2.groovy => changelog-2_0_2.groovy} (100%)
 rename grails-app/migrations/{changelog-2.0.3.groovy => changelog-2_0_3.groovy} (100%)
 create mode 100644 grails-app/views/layouts/login.gsp
 rename chado-schema-with-ontologies.sql.gz => scripts/chado-schema-with-ontologies.sql.gz (100%)
 create mode 100644 src/gwt/org/bbop/apollo/gwt/shared/ClientTokenGenerator.java
 create mode 100644 src/templates/war/web.xml
 rename wrapper/{grails-wrapper-runtime-2.4.3.jar => grails-wrapper-runtime-2.4.5.jar} (86%)
 create mode 100644 wrapper/grails-wrapper-runtime-2.5.4.jar
 delete mode 100644 wrapper/springloaded-1.2.0.RELEASE.jar
 create mode 100644 wrapper/springloaded-1.2.3.RELEASE.jar
 create mode 100644 wrapper/springloaded-1.2.4.RELEASE.jar

:copyApolloPlugin
:installJBrowsePlugins
installing jbrowse plugins  [WebApollo:[included:true], NeatHTMLFeatures:[included:true], NeatCanvasFeatures:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]
Evaluating plugin WebApollo=[included:true]
Plugin jbrowse-download/plugins/WebApollo exists and appears valid.
Evaluating plugin NeatHTMLFeatures=[included:true]
Plugin jbrowse-download/plugins/NeatHTMLFeatures exists and appears valid.
Evaluating plugin NeatCanvasFeatures=[included:true]
Plugin jbrowse-download/plugins/NeatCanvasFeatures exists and appears valid.
Evaluating plugin RegexSequenceSearch=[included:true]
Plugin jbrowse-download/plugins/RegexSequenceSearch exists and appears valid.
Evaluating plugin HideTrackLabels=[included:true]
Plugin jbrowse-download/plugins/HideTrackLabels exists and appears valid.
:handleJBrowse
:copy.apollo.plugin.webapp
:setup-jbrowse
[ant:exec] Result: 128
[ant:exec] Result: 1
[ant:echo] copying plugin to jbrowse git ...
[ant:echo] Will be setting up jbrowse
:setup-jbrowse FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':setup-jbrowse'.
> Unable to delete file /Users/mimiyxu/Apollo/web-app/jbrowse/plugins/WebApollo/apollo/js/View/Dialog/Help.js

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 41.802 secs
{
    "version": "4.4.0",
    "triggerid": "jsfilescopy",
    "disposition": "already_defined"
}"

I'd really like some help on what I should do. I have no background in computer science or anything relating to coding, so I have no idea what exactly failed and how I should proceed.

Thank you for reading! I'd really want to use Apollo!!

--- a/apollo
+++ b/apollo
@@ -16,6 +16,7 @@ function usage(){
     echo "run-app   <port>: Runs from current directory, but does not compile annotator panel (you have to run run-local once first).";
     echo "test:             Runs test-suite.";
     echo "debug:            Runs from current directory in debug mode (non-minimized javascript).";
+    echo "watchman:         Creates watchman daemon to autocopy plugin files (non-minimized javascript).";
     echo "compile:          Compiled the build.";
     echo "clean:            Removes class files.";
     echo "clean-all:        Removes class files and jbrowse files.";
@@ -93,10 +94,6 @@ function check_configs(){
     grails_executable=$(which grails)
     gradle_executable=$(which gradle)
     git_executable=$(which git)
-    watchman_executable=$(which watchman)
-    if ! [ -x "$watchman_executable" ] ; then
-        echo "Watchman not found. Install watchman to automatically update the client side plugins for apollo development modes"
-    fi
     if ! [ -x "$grails_executable" ] ; then
         if [ -f 'grailsw' ]; then
             echo "Grails not found using grailsw";
@@ -133,13 +130,17 @@ if [[ $1 == "devmode" ]];then
     check_configs
     $gradle_executable handleJBrowse copy-resources-dev devmode &
     $grails_executable -reloading run-app
-    if [ -x "$watchman_executable" ]; then $watchman_executable -- trigger $(PWD) jsfilescopy 'client/apollo/**/*.js' -- scripts/copy_client.sh; fi;
 elif [[ $1 == "run-local" ]];then
     check_configs
     if [[ $# == 2 ]]; then
         $gradle_executable handleJBrowse copy-resources-dev gwtc && $grails_executable -Dserver.port=$2 run-app
     else
         $gradle_executable handleJBrowse copy-resources-dev gwtc && $grails_executable run-app
+    fi
+elif [[ $1 == "watchman" ]]; then
+    watchman_executable=$(which watchman)
+    if ! [ -x "$watchman_executable" ] ; then
+        echo "Watchman not found. Install watchman to automatically update the client side plugins for apollo development modes"
     fi
     if [ -x "$watchman_executable" ]; then $watchman_executable -- trigger $(PWD) jsfilescopy 'client/apollo/**/*.js' -- scripts/copy_client.sh; fi;
 elif [[ $1 == "run-app" ]];then
@@ -149,7 +150,6 @@ elif [[ $1 == "run-app" ]];then
     else
         $gradle_executable handleJBrowse copy-resources-dev && $grails_executable run-app
     fi
-    if [ -x "$watchman_executable" ]; then $watchman_executable -- trigger $(PWD) jsfilescopy 'client/apollo/**/*.js' -- scripts/copy_client.sh; fi;
 elif [[ $1 == "debug" ]];then
     # TODO: feel like there is a better way to do this
     OLD_MAVEN_OPTS=$MAVEN_OPTS
@@ -160,7 +160,6 @@ elif [[ $1 == "debug" ]];then
     $grails_executable -reloading debug
     export MAVEN_OPTS=$OLD_MAVEN_OPTS
     unset OLD_MAVEN_OPTS
-    if [ -x "$watchman_executable" ]; then $watchman_executable -- trigger $(PWD) jsfilescopy 'client/apollo/**/*.js' -- scripts/copy_client.sh; fi;
 elif [[ $1 == "test" ]];then
     check_configs
     copy_configs

